edge-util/src/Microsoft.Azure.Devices.Edge.Util/metrics/prometheus.net/MetricsProvider.cs

```diff
@@ -13,17 +13,15 @@ public class MetricsProvider : IMetricsProvider
     {
         const string CounterNameFormat = "{0}_{1}_total";
         const string NameFormat = "{0}_{1}";
-        const string InstanceFile = "metrics_instance";
         readonly string namePrefix;
         readonly List<string> defaultLabelNames;
-        static readonly Lazy<string> instanceNumber = new Lazy<string>(GetInstanceNumber);
 
         public MetricsProvider(string namePrefix, string iotHubName, string deviceId)
         {
             this.namePrefix = Preconditions.CheckNonWhiteSpace(namePrefix, nameof(namePrefix));
             Preconditions.CheckNonWhiteSpace(iotHubName, nameof(iotHubName));
             Preconditions.CheckNonWhiteSpace(deviceId, nameof(deviceId));
-            this.defaultLabelNames = new List<string> { iotHubName, deviceId, instanceNumber.Value };
+            this.defaultLabelNames = new List<string> { iotHubName, deviceId, Guid.NewGuid().ToString() };
```

Comment:
I was deciding whether or not to have this injected into the constructor, but decided against it.  Let me know if you want me to change this.